### PR TITLE
Botup GTM read.me content change.

### PR DESCRIPTION
**Pull request by venkatkalluri-500apps**
Botup GTM read.me content change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,47 @@
...
-As your customer base grows from various locations, there's a wide scope for business growth. Bot up has the capability to scale your operations and make sure that you respond to your customers on time.
-Botup serves best with simple setup options itself. Use the advanced features to explore the all-in-one bot which is capable of handling all your social media handle responses at one go. Enjoy the additional benefit of capturing the leads and grow your network multi fold.
+A powerful platform to build chat bots for your teams on Facebook, Slack, WhatsApp, SMS, and Websites. Create flows and run a bot to respond swiftly to your consumers thereby enabling powerful automation campaigns. As your customer base grows from various locations, there's a wide scope for business growth. Bot up has the capability to scale your operations and make sure that you respond to your customers on time. Botup serves best with simple setup options itself. Use the advanced features to explore the all-in-one bot which is capable of handling all your social media handle responses at one go. Enjoy the additional benefit of capturing the leads and grow your network multi fold.
```
Comment (subramaniyamkolluri) [inline README.md]: Slack integration not there yet

Comment (venkatkalluri-500apps) [reply README.md]: Is there in apps tab but not implemented sir.

Comment (subramaniyamkolluri) [reply README.md]: remove it

Comment (venkatkalluri-500apps) [reply README.md]: Removed

**Comment by subramaniyamkolluri** [comment]
Fix the issue raise by the team

**Review by subramaniyamkolluri — APPROVED** [review]

**Pull request merged**